.github/workflows/pytest.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: Install dependencies
```

Comment:
Switch to conda.
The readme.md already advises to start with a conda env. 
(and in a pending pull request it will also suggest to use conda to install the requirments.txt)
Better test what we are advising

Reply:
I'd also add a requirements.test.txt with pytest, pytest-cov and in the future package for pep8 and other linting tools in there that we only need for this code quality action.
the build currently fails becase pytests is missing